e2e/test/provisioning/ProvisioningCertificateValidationE2ETest.cs

```diff
@@ -110,7 +112,7 @@ public async Task ProvisioningDeviceClient_RegisterAsyncInvalidServiceCertificat
 
         private async Task TestInvalidServiceCertificate(ProvisioningTransportHandler transport)
         {
-            string certificateSubject = $"{nameof(ProvisioningCertificateValidationE2ETest)}-{Guid.NewGuid()}";
+            string certificateSubject = "cert-" + Guid.NewGuid().ToString();
```

Comment:
Shorten the cert file name (long name caused flaky test)

Reply:
```suggestion
            string certificateSubject = $"cert-{Guid.NewGuid()}";
```

Reply:
Updated with latest commit.